package.json

```diff
@@ -90,6 +90,7 @@
     "uuid": "^9.0.1",
     "verdaccio": "^5.29.2",
     "vite": "^5.2.6",
+    "vite-plugin-dts": "^3.8.3",
     "vite-plugin-eslint": "^1.8.1",
     "vite-tsconfig-paths": "^4.3.2",
```

Comment:
Ugh, so many of these Vite plugins are maintained by a single individual. Make me nervous.